skimage/future/__init__.py

```diff
@@ -1,7 +1,9 @@
 """Functionality with an experimental API. Although you can count on the
 functions in this package being around in the future, the API may change with
 any version update **and will not follow the skimage two-version deprecation
-path**. Therefore, use the functions herein with care, and do not use them in
+path**.
+
+Therefore, use the functions herein with care, and do not use them in
```

Comment:
Hey @ana42742, thanks for picking this up. I think my description in #6761 was not clear enough. :) The "short description" or short summary (the term used by the [numpydoc style guide](https://numpydoc.readthedocs.io/en/latest/format.html#short-summary)) is only the first line. So that's were I meant the blank line to go. 

And I would suggest to put the entire text following behind that into a [warning directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-warning). Does that help?